settings/settings-module.php

```diff
@@ -145,62 +170,62 @@ public function __construct( &$polylang, $args ) {
 			'save'   => sprintf( '<button type="button" class="button button-primary save">%s</button>', esc_html__( 'Save Changes', 'polylang' ) ),
 		);
 
-		// Ajax action to save options
+		// Ajax action to save options.
 		add_action( 'wp_ajax_pll_save_options', array( $this, 'save_options' ) );
 	}
 
 	/**
-	 * Tells if the module is active
+	 * Tells if the module is active.
 	 *
 	 * @since 1.8
 	 *
 	 * @return bool
 	 */
 	public function is_active() {
-		return empty( $this->active_option ) || ! empty( $this->options[ $this->active_option ] );
+		return 'none' === $this->active_option || 'preview' !== $this->active_option && ! empty( $this->options[ $this->active_option ] );
```

Comment:
Could you add parentheses? I don't like relying on operator precedence.

Reply:
Done in [daeb7eb](https://github.com/polylang/polylang/pull/1420/commits/daeb7eb86853195c76675b76bd5141c96be20faa).